packages/scandipwa/src/component/WishlistItem/WishlistItem.style.scss

```diff
@@ -175,6 +176,14 @@
         }
     }
 
+    &-Remove {
+        @include mobile {
+            &_withReview[aria-label='Remove'] {
```

Comment:
Is it necessary to raise the CSS selector specificity by putting the attribute with value?
If yes, please use the `mods` prop in order to add an additional, more specific, class name.

Reply:
BTW, this will not work with other languages, since "Remove" is put to the translation function.
```javascript
aria-label={ __('Remove') }
```